src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,11 +5,26 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
Don't begin class implementation with an empty line. Remove the extra lines between the package declaration and the class declaration.